terraform/providers/google/go.mod

```diff
@@ -1,41 +1,41 @@
 module github.com/openshift/installer/terraform/providers/google
 
-go 1.18
+go 1.19
```

Comment:
The installer uses go version 1.20 [here](https://github.com/openshift/installer/blob/master/go.mod#L3) Should it not be the same here as well?

Reply:
Yeah, but the provider is still using 1.19, so thought to keep the same. Should I update the go version to 1.20 same as installer?

Reply:
@patrickdillon please suggest.

Reply:
@jhixson74 is working on updating the go version of terraform providers to 1.20.

Reply:
@bharath-b-rh The Golang versions are supposed to be backward-compatible, so any version >= 1.19 should be safe to use.

Reply:
Thank you @r4f4 , I will make the changes for google provider.

Reply:
suggestion incorporated.